app/src/main/java/bg/zahov/app/util/Extensions.kt

```diff
@@ -21,6 +21,8 @@ import bg.zahov.app.ui.exercise.ExerciseAdapterWrapper
 import bg.zahov.app.ui.workout.add.ExerciseSetAdapterExerciseWrapper
 import bg.zahov.app.ui.workout.add.ExerciseSetAdapterSetWrapper
 import bg.zahov.fitness.app.R
+import com.github.mikephil.charting.charts.BarChart
```

Comment:
It is generally a bad practice to have one file containing all utility functions. They blow up out of proportions.
Please add the new function to its corresponding place of use or to a separate file.